WEB_SERVICE/WEB/_HIGH/4XX_BYPASS/meta.json

```diff
@@ -0,0 +1,47 @@
+{
+    "risk_rating": "high",
+    "short_description": "Insecure Authorization Restriction vulnerability allows an attacker to bypass access controls by manipulating HTTP requests to gain access to restricted resources or perform actions they are not supposed to.",
+    "references": {
+        "wikipedia": "https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors",
+        "Hacktricks": "https://book.hacktricks.xyz/network-services-pentesting/pentesting-web/403-and-401-bypasses",
+        "cheatsheetseries.owasp": "https://cheatsheetseries.owasp.org/assets/REST_Security_Cheat_Sheet_Bypassing_VBAAC_with_HTTP_Verb_Tampering.pdf"
+    },
+    "title": "Insecure Authorization Restriction Vulnerability",
```

Comment:
Can you please change the folder of the KB to something like

`WEB_SERVICE/WEB/_HIGH/INSECURE_AUTHORIZATION_RESTRICTION`

Reply:
Done!